src/System.Devices.I2c/II2cConnectionSettings.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Devices.I2c
+{
+    internal interface II2cConnectionSettings
```

Comment:
Why do we need this interface (and other interfaces in this assembly)? It seems like there is a single implementation for them and we don't have any APIs that actually consume these interfaces. In .NET, we add abstractions in cases when multiple types implement an abstractions and also some APIs consume (take as parameter, for example) the same abstraction.

Also, we prefer classes over interfaces for abstractions. We only use interfaces when we need types to implement more than one abstraction (i.e. multiple-inheritance).